src/Scaffold/Console/plugin/plugin.stub

```diff
@@ -42,6 +42,54 @@ class Plugin extends PluginBase
     {
 
     }
+    
+    /**
+     * Registers scheduled tasks that are executed on a regular basis.
+     *
+     * @param string $schedule
```

Comment:
Hi @Samuell1, the register method gives the Schedule object instead of string :), see octobercms/october#4871. Could you change that?